### PR TITLE
Changed CSV Type to Schema Type

### PR DESCRIPTION
**Pull request by anojperera**
This change will xml based model driven CSV Content Type to Schema driven type. 
This will give us more control and flexibility.

Acceptance Criteria
User is able to create CSV Content Type
Add a title and upload a CSV.

There's no validation done for the content type at the moment.

### CODE REVIEW

File: plone-5/instance/src/ukstats.csv_type/src/ukstats/csv_type/csv.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from ukstats.csv_type import _
+from plone.supermodel import model
+from plone.namedfile.field import NamedBlobFile
+from zope import schema
+
+
+class ICSV(model.Schema):
+    """A conference program. Programs can contain Sessions."""
```
Comment (goofballLogic) [inline plone-5/instance/src/ukstats.csv_type/src/ukstats/csv_type/csv.py:8]: what's this?

File: plone-5/instance/src/ukstats.csv_type/src/ukstats/csv_type/csv.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    csv = NamedBlobFile(
+        title=_(u'csv'),
```
Comment (goofballLogic) [inline plone-5/instance/src/ukstats.csv_type/src/ukstats/csv_type/csv.py:10]: the title looks a bit weird in the UI (esp. the Volto one)?
Screenshot 2022-03-22 at 17 00 42
Screenshot 2022-03-22 at 17 00 57

File: plone-5/instance/src/ukstats.csv_type/src/ukstats/csv_type/profiles/default/types/csv_type.xml [hunk 3]
```diff
@@ -29,9 +22,9 @@
   <!-- Schema, class and security -->
   <property name="add_permission">ukstats.csv_type.AddCSVType</property>
   <property name="klass">plone.dexterity.content.Container</property>
-  <property name="model_file">ukstats.csv_type.content:csv_type.xml</property>
```
Comment (goofballLogic) [inline plone-5/instance/src/ukstats.csv_type/src/ukstats/csv_type/profiles/default/types/csv_type.xml:32]: the XML file is still in the folder - is that expected?

**Review by goofballLogic — COMMENTED** [review]
Few questions

File: plone-5/instance/src/ukstats.csv_type/src/ukstats/csv_type/profiles/default/types/csv_type.xml (hunk 3 above)
Comment (anojperera) [reply plone-5/instance/src/ukstats.csv_type/src/ukstats/csv_type/profiles/default/types/csv_type.xml:32]: I've deleted the model xml

**Review by goofballLogic — APPROVED** [review]
looks good

**Pull request merged**